src/FillArrays.jl

```diff
@@ -1,10 +1,12 @@
+""" `FillArrays` module to lazily represent matrices with a single value """
```

Comment:
Delete this: I've never seen a package have a doc string before so this is not standard.

Reply:
Ok, I will delete it.
Just for the record this is standard enough that it is documented in the manual:
https://docs.julialang.org/en/v1/manual/documentation/index.html#Modules-1
but this was my first time trying it out - just trying to be helpful...

Reply:
Ok in that case leave it (never seen it before...)

Reply:
Ok, I put it back.
(I haven't seen it much either and a good README for a package seems much more important.)